builder/azure/arm/builder.go

```diff
@@ -248,7 +248,9 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 		}
 
 		if len(b.config.SharedGalleryDestination.SigDestinationTargetRegions) == 0 {
-			return nil, errors.New("no target destination region specified for the Shared Image Gallery; use target_region to specify at least the primary destination region for storing the image version.")
+			ui.Say("WARNING! No target region was set for the Shared Image Gallery. Adding %q as the primary target region for now but this may change in the future.")
```

Comment:
You forgot to add a `Sprintf` here to expand the `%q` I believe?

Reply:
No I didn't 😉 - good catch.